{cc,bintools}-wrapper: Prevent `NIX_HARDENING_ENABLE` clobbers

Ericson2314 opened the pull request:
We've been doing the "suffix salt" for this variable like the others, but we never bothered setting role-specific variables in the setup hook.

Translation from obscure gobbledygoop: if you are using something like `buildPackage.stdednv.cc`, it might want (for the build platform) more formatting than the host platform supports, and then all hell breaks loose. Now the build platform-targetting compiler will use `NIX_HARDENING_ENABLE_FOR_BUILD` and there will be no conflict.

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

Ericson2314 committed:
{cc,bintools}-wrapper: Prevent `NIX_HARDENING_ENABLE` clobbers

We've been doing the "suffix salt" for this variable like the others,
but we never bothered setting role-specific variables in the setup hook.

Translation from obscure gobbledygoop above: if you are using something
like `buildPackage.stdednv.cc`, it might want (for the build platform)
more formatting than the host platform supports, and then all hell
breaks loose. Now the build platform-targetting compiler will use
`NIX_HARDENING_ENABLE_FOR_BUILD` and there will be no conflict.

risicle commented:
Any changes needed in `mkDerivation` too?

Ericson2314 commented:
@risicle That would be a good idea, but that should come separately.

RossComputerGuy commented:
Just got around to testing if this PR builds the Linux kernel on aarch64-linux for LLVM. I seem to run into a weird failure:
```
error:
       … in the condition of the assert statement
         at /nix/store/981jbarpm02012h7z33frfv0hm5nj526-source/lib/customisation.nix:417:9:
          416|       drvPath =
          417|         assert condition;
             |         ^
          418|         drv.drvPath;

       … while evaluating a branch condition
         at /nix/store/981jbarpm02012h7z33frfv0hm5nj526-source/pkgs/stdenv/generic/check-meta.nix:504:6:
          503|       inherit (validity) valid;
          504|   in if validity ? handled then validity else validity // {
             |      ^
          505|       # Throw an error if trying to evaluate a non-valid derivation

       … while evaluating the option `settings':

       … while evaluating definitions from `pkgs/os-specific/linux/kernel/common-config.nix':

       (stack trace truncated; use '--show-trace' to show the full, detailed trace)

       error: undefined variable 'isLoongarch64'
       at /nix/store/981jbarpm02012h7z33frfv0hm5nj526-source/pkgs/os-specific/linux/kernel/common-config.nix:512:51:
          511|           || (lib.versionAtLeast version "6.2" && isAarch64 && !stdenv.cc.isClang)
          512|           || (lib.versionAtLeast version "6.5" && isLoongarch64 && !stdenv.cc.isClang)
             |                                                   ^
          513|           || (lib.versionAtLeast version "6.10" && isRiscV64 && !stdenv.cc.isClang)
```

I also tried removing that line and got a build failure in coreutils. This was done inside a `nixpkgs-review` shell.

risicle commented:
I'm not sure how edits to three bash files would cause a nix eval error - guessing this is a merge/rebase conflict somewhere?

RossComputerGuy commented:
Maybe but `nixpkgs-review` merges right? Or does it only do a merge test?

snue committed:
linuxPackages.ply: 2.1.1+ -> 2.4.0

r-ryantm committed:
cli11: 2.5.0 -> 2.6.1

r-ryantm committed:
thin-provisioning-tools: 1.3.0 -> 1.3.1

r-ryantm committed:
scap-security-guide: 0.1.78 -> 0.1.79

r-ryantm committed:
acme-sh: 3.1.1 -> 3.1.2

booxter committed:
ovn: run scapy based tests

With scapy in python import path, additional test cases that use it will
be executed during test run.

booxter committed:
ovn: fix command presence detection during checkPhase

The test suite uses `which` to detect if auxiliary dependencies like
scapy or tcpdump are installed on the system, and skips the tests using
these commands if not.

Problem is when `which` is missing, all such commands are detected as
missing, so some tests that could be safely run are unnecessarily
skipped.

Note: upstream should probably not swallow the error from missing
`which` and instead enforce its presence during build. Or better: use
`command -v` that is always present.

cmm committed:
rassumfrassum: init at 0.3.3

Build with checks disabled, at least for the time being.

r-ryantm committed:
xed: 2025.06.08 -> 2025.12.14

r-ryantm committed:
pipenv: 2025.0.4 -> 2026.0.3

r-ryantm committed:
fluidd: 1.34.4 -> 1.36.2

fabaff committed:
python313Packages.aioamazondevices: 11.0.2 -> 11.1.1

Diff: https://github.com/chemelli74/aioamazondevices/compare/v11.0.2...v11.1.1

Changelog: https://github.com/chemelli74/aioamazondevices/blob/v11.1.1/CHANGELOG.md

bbenno committed:
whatsapp-chat-exporter: 0.12.1 → 0.13.0

Adda0 committed:
atuin-desktop: 0.2.11 -> 0.2.19

dotlambda committed:
python3Packages.pyjvcprojector: 1.1.3 -> 2.0.0

Diff: https://github.com/SteveEasley/pyjvcprojector/compare/v1.1.3...v2.0.0

Changelog: https://github.com/SteveEasley/pyjvcprojector/releases/tag/v2.0.0

LordGrimmauld committed:
qutebrowser-qt5: drop

r-ryantm committed:
pyenv: 2.6.14 -> 2.6.20

fabaff committed:
python313Packages.pynina: 0.3.6 -> 1.0.2

Changelog: https://gitlab.com/DeerMaximum/pynina/-/releases/v1.0.2

fabaff committed:
python313Packages.pynina: migrate to finalAttrs

r-ryantm committed:
healthchecks: 3.11.2 -> 4.0

r-ryantm committed:
python3Packages.pyicloud: 2.2.0 -> 2.3.0

r-ryantm committed:
python3Packages.weheat: 2025.12.24 -> 2026.1.25

r-ryantm committed:
python3Packages.velbus-aio: 2026.1.1 -> 2026.1.4

Mic92 committed:
glibc: fix darwin-cross-build.patch for glibc 2.42

The darwin-cross-build.patch failed to apply when cross-compiling glibc.

Regenerate the patch against glibc 2.42 sources. Also extend the
patch to cover the 'check xcheck test' target which was previously
unpatched but needs the same ulimit and AR fixes for consistency.

Tested: nix build --file . pkgsCross.aarch64-multiplatform.glibc
Tested: nix build --file . pkgsCross.gnu64.glibc

r-ryantm committed:
git-credential-aol: 5.5.2 -> 5.6.2

markhakansson committed:
mdbook-plantuml: 0.8.0 -> 2.0.0

eclairevoyant committed:
signal-desktop-bin: 7.86.0 -> 7.88.0

iedame committed:
bbedit: 15.5.3 -> 15.5.4

fabaff committed:
python314Packages.python-roborock: 4.2.1 -> 4.8.0

Diff: https://github.com/Python-roborock/python-roborock/compare/v4.2.1...v4.8.0

Changelog: https://github.com/Python-roborock/python-roborock/blob/v4.8.0/CHANGELOG.md

fabaff committed:
python314Packages.python-roborock: migrate to finalAttrs

1183519e committed:
ptouch-print: 1.5-unstable-2024-02-11 -> 1.7-unstable-2026-01-23

Uzlkav committed:
pywal16: 3.18.13 -> 3.18.14

debtquity committed:
nixos/nextcloud: update test import

* `stalwart` module refactored as part of https://github.com/NixOS/nixpkgs/pull/481815 resulted in rename of `../stalwart/stalwart-mail-config.nix` to `../stalwart/stalwart-config.nix`

dotlambda committed:
python3Packages.todoist-api-python: 2.1.7 -> 3.2.1

Diff: https://github.com/Doist/todoist-api-python/compare/v2.1.7...v3.2.1

Changelog: https://github.com/Doist/todoist-api-python/blob/v3.2.1/CHANGELOG.md

Jappie3 committed:
webmacs: drop

dotlambda committed:
immich: 2.5.2 -> 2.5.3

Diff: https://github.com/immich-app/immich/compare/v2.5.2...v2.5.3

Changelog: https://github.com/immich-app/immich/releases/tag/v2.5.3

dotlambda committed:
home-assistant: 2026.1.3 -> 2026.2.0

Diff: https://github.com/home-assistant/core/compare/2026.1.3...2026.2.0

Changelog: https://www.home-assistant.io/blog/2026/02/04/release-20262/

dotlambda committed:
python3Packages.aiosyncthing: 0.6.3 -> 0.7.1

Diff: https://github.com/zhulik/aiosyncthing/compare/v0.6.3...v0.7.1

Changelog: https://github.com/zhulik/aiosyncthing/releases/tag/v0.7.1

dotlambda committed:
python3Packages.aiotractive: 0.7.0 -> 1.0.0

Diff: https://github.com/zhulik/aiotractive/compare/v0.7.0...v1.0.0

Changelog: https://github.com/zhulik/aiotractive/releases/tag/v1.0.0

dotlambda committed:
python3Packages.async-upnp-client: 0.46.1 -> 0.46.2

Diff: https://github.com/StevenLooman/async_upnp_client/compare/0.46.1...0.46.2

Changelog: https://github.com/StevenLooman/async_upnp_client/blob/0.46.2/CHANGES.rst

dotlambda committed:
python3Packages.python-didl-lite: 1.4.1 -> 1.5.0

Diff: https://github.com/StevenLooman/python-didl-lite/compare/1.4.1...1.5.0

Changelog: https://github.com/StevenLooman/python-didl-lite/blob/1.5.0/CHANGES.rst

dotlambda committed:
python3Packages.bleak: 2.0.0 -> 2.1.1

Diff: https://github.com/hbldh/bleak/compare/v2.0.0...v2.1.1

Changelog: https://github.com/hbldh/bleak/blob/v2.1.1/CHANGELOG.rst

dotlambda committed:
python3Packages.compit-inext-api: 0.5.0 -> 0.8.0

Diff: https://github.com/Przemko92/compit-inext-api/compare/0.5.0...0.8.0

Changelog: https://github.com/Przemko92/compit-inext-api/releases/tag/0.8.0

dotlambda committed:
python3Packages.hass-nabucasa: 1.7.0 -> 1.12.0

Diff: https://github.com/nabucasa/hass-nabucasa/compare/1.7.0...1.12.0

Changelog: https://github.com/NabuCasa/hass-nabucasa/releases/tag/1.12.0

dotlambda committed:
python3Packages.influxdb-client: 1.49.0 -> 1.50.0

Diff: https://github.com/influxdata/influxdb-client-python/compare/v1.49.0...v1.50.0

Changelog: https://github.com/influxdata/influxdb-client-python/blob/v1.50.0/CHANGELOG.md

dotlambda committed:
python3Packages.onedrive-personal-sdk: 0.1.0 -> 0.1.1

Diff: https://github.com/zweckj/onedrive-personal-sdk/compare/v0.1.0...v0.1.1

Changelog: https://github.com/zweckj/onedrive-personal-sdk/releases/tag/v0.1.1

dotlambda committed:
python3Packages.pyfirefly: 0.1.9 -> 0.1.13

Diff: https://github.com/erwindouna/pyfirefly/compare/v0.1.9...v0.1.13

Changelog: https://github.com/erwindouna/pyfirefly/releases/tag/v0.1.13

dotlambda committed:
python3Packages.python-bsblan: 3.1.6 -> 4.1.0

Diff: https://github.com/liudger/python-bsblan/compare/v3.1.6...v4.1.0

Changelog: https://github.com/liudger/python-bsblan/releases/tag/v4.1.0

dotlambda committed:
python3Packages.python-xbox: 0.1.2 -> 0.1.3

Diff: https://github.com/tr4nt0r/python-xbox/compare/v0.1.2...v0.1.3

Changelog: https://github.com/tr4nt0r/python-xbox/releases/tag/v0.1.3

dotlambda committed:
python3Packages.pyvesync: 3.4.0 -> 3.4.1

Diff: https://github.com/webdjoe/pyvesync/compare/3.4.0...3.4.1

Changelog: https://github.com/webdjoe/pyvesync/releases/tag/3.4.1

dotlambda committed:
python3Packages.librehardwaremonitor-api: 1.6.0 -> 1.8.4

Diff: https://github.com/Sab44/librehardwaremonitor-api/compare/v1.6.0...v1.8.4

THS-on committed:
spdk: 25.05 -> 26.01

davidweisse committed:
maintainers: add davidweisse

davidweisse committed:
json-patch: init at 5.9.11

dotlambda committed:
python3Packages.python-otbr-api: 2.7.1 -> 2.8.0

Diff: https://github.com/home-assistant-libs/python-otbr-api/compare/2.7.1...2.8.0

Changelog: https://github.com/home-assistant-libs/python-otbr-api/releases/tag/2.8.0

dotlambda committed:
python3Packages.zha: 0.0.84 -> 0.0.89

Diff: https://github.com/zigpy/zha/compare/0.0.84...0.0.89

Changelog: https://github.com/zigpy/zha/releases/tag/0.0.89

dotlambda committed:
python3Packages.zigpy: 0.90.0 -> 0.91.5

Diff: https://github.com/zigpy/zigpy/compare/0.90.0...0.91.5

Changelog: https://github.com/zigpy/zigpy/releases/tag/0.91.5

dotlambda committed:
python3Packages.zigpy-znp: 0.14.2 -> 0.14.3

Diff: https://github.com/zigpy/zigpy-znp/compare/v0.14.2...v0.14.3

Changelog: https://github.com/zigpy/zigpy-znp/releases/tag/v0.14.3

dotlambda committed:
python3Packages.zha-quirks: 0.0.152 -> 0.0.156

Diff: https://github.com/zigpy/zha-device-handlers/compare/0.0.152...0.0.156

Changelog: https://github.com/zigpy/zha-device-handlers/releases/tag/0.0.156

r-ryantm committed:
gnome-commander: 1.18.4 -> 1.18.5

r-ryantm committed:
python3Packages.cpyparsing: 2.4.7.2.4.1 -> 2.4.7.2.4.2

khaneliman committed:
luaPackages: update on 2026-02-04

Signed-off-by: Austin Horstman <khaneliman12@gmail.com>

khaneliman committed:
luaPackages.rocks-nvim: 2.47.2-1 -> 2.47.3-1

dotlambda committed:
home-assistant.frontend: 20260107.2 -> 20260128.6

Changelog: https://github.com/home-assistant/frontend/releases/tag/20260128.6

dotlambda committed:
home-assistant.intents: 2026.1.6 -> 2026.1.28

Diff: https://github.com/OHF-Voice/intents-package/compare/2026.1.6...2026.1.28

Changelog: https://github.com/OHF-Voice/intents-package/releases/tag/2026.1.28

drupol committed:
hedgedoc: 1.10.4 -> 1.10.6

Fix security vulnerabilities:
 - https://github.com/hedgedoc/hedgedoc/security/advisories/GHSA-x74j-jmf9-534w
 - https://github.com/hedgedoc/hedgedoc/security/advisories/GHSA-672m-p72w-gw28

dotlambda committed:
python3Packages.aiohomematic: 2026.1.54 -> 2026.2.0

Diff: https://github.com/SukramJ/aiohomematic/compare/2026.1.54...2026.2.0

Changelog: https://github.com/SukramJ/aiohomematic/blob/2026.2.0/changelog.md

dotlambda committed:
home-assistant-custom-components.homematicip_local: 2.2.3 -> 2.2.4

Diff: https://github.com/SukramJ/custom_homematic/compare/2.2.3...2.2.4

Changelog: https://github.com/SukramJ/custom_homematic/blob/2.2.4/changelog.md

khaneliman committed:
luaPackages.rocks-nvim: 2.47.3-1 -> 2.47.4-1

dotlambda committed:
home-assistant.python.pkgs.pytest-homeassistant-custom-component: 0.13.308 -> 0.13.313

Diff: https://github.com/MatthewFlamm/pytest-homeassistant-custom-component/compare/0.13.308...0.13.313

Changelog: https://github.com/MatthewFlamm/pytest-homeassistant-custom-component/blob/0.13.313/CHANGELOG.md

khaneliman committed:
luaPackages.rocks-nvim: disable check on darwin

Weird transient failures.

Signed-off-by: Austin Horstman <khaneliman12@gmail.com>

khaneliman committed:
luaPackages.rocks-dev-nvim: disable check on darwin

Signed-off-by: Austin Horstman <khaneliman12@gmail.com>

dotlambda committed:
home-assistant-custom-components.mitsubishi: 0.5.3 -> 0.5.4

Diff: https://github.com/pymitsubishi/homeassistant-mitsubishi/compare/v0.5.3...v0.5.4

Changelog: https://github.com/pymitsubishi/homeassistant-mitsubishi/releases/tag/v0.5.4

dotlambda committed:
home-assistant-custom-components.powercalc: 1.20.3 -> 1.20.4

Diff: https://github.com/bramstroker/homeassistant-powercalc/compare/v1.20.3...v1.20.4

Changelog: https://github.com/bramstroker/homeassistant-powercalc/releases/tag/v1.20.4

r-ryantm committed:
gatekeeper: 3.21.0 -> 3.21.1

r-ryantm committed:
rauthy: 0.34.2 -> 0.34.3

r-ryantm committed:
grpc-gateway: 2.27.6 -> 2.27.7

dotlambda committed:
python3Packages.homeassistant-stubs: 2026.1.3 -> 2026.2.0

Diff: https://github.com/KapJI/homeassistant-stubs/compare/2026.1.3...2026.2.0

Changelog: https://github.com/KapJI/homeassistant-stubs/releases/tag/2026.2.0

dotlambda committed:
home-assistant-custom-components.epex_spot: 4.0.0 -> 4.1.0

Diff: https://github.com/mampfes/ha_epex_spot/compare/4.0.0...4.1.0

Changelog: https://github.com/mampfes/ha_epex_spot/releases/tag/4.1.0

dotlambda committed:
home-assistant-custom-components.midea_ac: 2026.1.1 -> 2026.2.0

Diff: https://github.com/mill1000/midea-ac-py/compare/2026.1.1...2026.2.0

Changelog: https://github.com/mill1000/midea-ac-py/releases/tag/2026.2.0

dotlambda committed:
home-assistant-custom-components.polaris-mqtt: 1.0.13 -> 1.1.0

Diff: https://github.com/samoswall/polaris-mqtt/compare/v1.0.13...v1.1.0

dotlambda committed:
home-assistant-custom-components.sensi: 1.4.8 -> 2.1.2

Diff: https://github.com/iprak/sensi/compare/v1.4.8...v2.1.2

Changelog: https://github.com/iprak/sensi/releases/tag/v2.1.2

dotlambda committed:
home-assistant-custom-components.tuya_local: 2026.1.1 -> 2026.1.2

Diff: https://github.com/make-all/tuya-local/compare/2026.1.1...2026.1.2

Changelog: https://github.com/make-all/tuya-local/releases/tag/2026.1.2

dotlambda committed:
python314Packages.reactivex: disable

LRFLEW committed:
nixos/terraria: add package option

dotlambda committed:
home-assistant-custom-lovelace-modules.card-mod: 4.1.0 -> 4.2.0

Diff: https://github.com/thomasloven/lovelace-card-mod/compare/v4.1.0...v4.2.0

dotlambda committed:
home-assistant-custom-lovelace-modules.custom-sidebar: 11.1.0 -> 12.0.0

Diff: https://github.com/elchininet/custom-sidebar/compare/v11.1.0...v12.0.0

Changelog: https://github.com/elchininet/custom-sidebar/releases/tag/v12.0.0

dotlambda committed:
home-assistant-custom-lovelace-modules.universal-remote-card: 4.9.7 -> 4.9.8

Diff: https://github.com/Nerwyn/android-tv-card/compare/4.9.7...4.9.8

dotlambda committed:
home-assistant-custom-lovelace-modules.material-you-utilities: 2.0.28 -> 2.0.31

Diff: https://github.com/Nerwyn/material-you-utilities/compare/2.0.28...2.0.31

Changelog: https://github.com/Nerwyn/material-you-utilities/releases/tag/2.0.31

philipmw committed:
erigon: 3.0.4 -> 3.3.7

Also add the version check hook.

r-ryantm committed:
synapse-admin-etkecc: 0.11.1-etke52 -> 0.11.1-etke53

r-ryantm committed:
twm: 0.12.3 -> 0.13.0

r-ryantm committed:
grpc-health-probe: 0.4.44 -> 0.4.45

kachick committed:
whosthere: fix version display

Switched to `versionCheckHook` to prevent similar issues,
as we easily miss running `passthru.tests`.

kachick committed:
whosthere: add shell completions

THS-on committed:
maintainers: add THS-on

THS-on committed:
spdk: set THS-on as maintainer

Mic92 committed:
spdk: 25.05 -> 26.01 (#486858)

c6rg0 committed:
uv: 0.9.29 -> 0.9.30

yuyuyureka committed:
ptouch-print: 1.5-unstable-2024-02-11 -> 1.7-unstable-2026-01-23 (#485296)

SigmaSquadron committed:
nixos/terraria: add package option (#487125)

wegank committed:
scap-security-guide: 0.1.78 -> 0.1.79 (#466025)

wegank committed:
thin-provisioning-tools: 1.3.0 -> 1.3.1 (#463052)

happysalada committed:
erigon: 3.0.4 -> 3.3.7 (#487155)

Mic92 committed:
glibc: fix darwin-cross-build.patch for glibc 2.42 (#484107)

nixpkgs-ci[bot] committed:
rauthy: 0.34.2 -> 0.34.3 (#487112)

wegank committed:
fluidd: 1.34.4 -> 1.36.2 (#461106)

wegank committed:
healthchecks: 3.11.2 -> 4.0 (#465813)

wegank committed:
acme-sh: 3.1.1 -> 3.1.2 (#466370)

GaetanLepage committed:
uv: 0.9.29 -> 0.9.30 (#487215)

Defelo committed:
synapse-admin-etkecc: 0.11.1-etke52 -> 0.11.1-etke53 (#487146)

wegank committed:
git-credential-aol: 5.5.2 -> 5.6.2 (#474743)

nixpkgs-ci[bot] committed:
grpc-gateway: 2.27.6 -> 2.27.7 (#487114)

nixpkgs-ci[bot] committed:
twm: 0.12.3 -> 0.13.0 (#487167)

nixpkgs-ci[bot] committed:
grpc-health-probe: 0.4.44 -> 0.4.45 (#487181)

wegank committed:
pyenv: 2.6.14 -> 2.6.20 (#469238)

wegank committed:
pipenv: 2025.0.4 -> 2026.0.3 (#468824)

wegank committed:
xed: 2025.06.08 -> 2025.12.14 (#466182)

drupol committed:
hedgedoc: 1.10.4 -> 1.10.6 (#487033)

jopejoe1 committed:
webmacs: drop (#486693)

SuperSandro2000 committed:
bbedit: 15.5.3 -> 15.5.4 (#485115)

SuperSandro2000 committed:
cli11: 2.5.0 -> 2.6.1 (#453931)

khaneliman committed:
luaPackages: update on 2026-02-04 (#487007)

adamcstephens committed:
ovn: run scapy based tests (#468757)

SuperSandro2000 committed:
pywal16: 3.18.13 -> 3.18.14 (#485814)

dotlambda committed:
immich: 2.5.2 -> 2.5.3 (#486783)

Aleksanaa committed:
gnome-commander: 1.18.4 -> 1.18.5 (#486975)

katexochen committed:
json-patch: init at 5.9.11 (#486944)

r-ryantm committed:
python3Packages.jupyter-book: 2.1.1 -> 2.1.2

teutat3s committed:
signal-desktop-bin: 7.86.0 -> 7.88.0 (#485099)

SuperSandro2000 committed:
gatekeeper: 3.21.0 -> 3.21.1 (#487093)

GaetanLepage committed:
python3Packages.jupyter-book: 2.1.1 -> 2.1.2 (#487277)

K900 committed:
kdePackages: Gear 25.12.1 -> 25.12.2

fabianhjr committed:
python3Packages.cpyparsing: 2.4.7.2.4.1 -> 2.4.7.2.4.2 (#486995)

K900 committed:
kdePackages: Gear 25.12.1 -> 25.12.2 (#487306)

dotlambda committed:
home-assistant: 2026.1.3 -> 2026.2.0 (#486830)

nixpkgs-ci[bot] committed:
mdbook-plantuml: 0.8.0 -> 2.0.0 (#486919)

dotlambda committed:
qutebrowser-qt5: drop (#487217)

SuperSandro2000 committed:
whatsapp-chat-exporter: 0.12.1 → 0.13.0 (#483588)

xanderio committed:
linuxPackages.ply: 2.1.1+ -> 2.4.0 (#404981)

andir committed:
rassumfrassum: init at 0.3.3 (#470255)

SuperSandro2000 committed:
nixos/nextcloud: fix test import (#486147)

nixpkgs-ci[bot] committed:
whosthere: fix version display, add shell completions (#487197)

SuperSandro2000 committed:
atuin-desktop: 0.2.11 -> 0.2.19 (#479590)
